Source/Example Measurements/Game Of Life/Game Of Life.lvproj

```diff
@@ -1,5 +1,8 @@
 ﻿<?xml version='1.0' encoding='UTF-8'?>
 <Project Type="Project" LVVersion="21008000">
+	<Property Name="NI.LV.All.SaveVersion" Type="Str">Editor version</Property>
```

Comment:
This is a good observation. I don't believe I explicitly set the version to "21.0" via the project settings so LV seems to have made this edit on project load. Playing around with things locally a bit more, LV seems to:
- Have some non-determinism when loading old projects in the presence of a .lvversion file as to whether this property will get set to "Editor Version" or the version from the .lvversion file.
- Give precedence to the save version in the project file if it is set to a specific version rather than the .lvversion file.
- Give precedence to the version in the .lvversion file if the save version in the project file is set to "Editor version"
- Have some inconsistency in how it displays the save version next to the project item in the project explorer window:
  - It **does not** display a version when the project is set to "Editor version" even though it is using the version from the .lvversion file which does not match the current editor version. Opening VIs in this state still displays the correct version from the .lvversion file. 
  - It **does not** display a version when the project save version is explicitly set to the same version as the editor version.
  - It **does** display a version when the project save version is explicitly set to any specific version that does not match the current editor version.

Given these observations, I am modifying all current projects in the repo to specify a save version of "Editor version" so that they will track the version in the .lvversion file. As we add new projects to the repo, we will need to be mindful to explicitly set this to "Editor version" or it will not track the .lvversion file.

Reply:
@jasonmreding can you add a screenshot of where this setting is changed?